test/specs/components/Chat/ChatMessage-test.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react'
+import { isConformant, implementsShorthandProp } from 'test/specs/commonTests'
+import { mountWithProvider } from '../../../utils'
+
+import ChatMessage from 'src/components/Chat/ChatMessage'
+import Avatar from 'src/components/Avatar'
+
+describe('ChatMessage', () => {
+  isConformant(ChatMessage)
+  implementsShorthandProp(ChatMessage)('avatar', Avatar, { mapsValueToProp: 'name' })
+
+  describe('avatar', () => {
+    it('creates an Avatar component when the avatar shorthand is provided', () => {
+      const name = 'AB'
```

Comment:
please, feel free to ignore this comment, but I would rather expect to see there something that reads like a name :) once again, may agree that this is rather a personal expectation

Reply:
np, done :)